.gitlab-ci.yml

```
@@ -166,3 +168,19 @@ publish:s3:legacy:mender-client:latest:
         aws s3api put-object-acl --acl public-read --bucket ${S3_BUCKET_NAME}
           --key latest/${S3_BUCKET_SUBPATH}/${arch}/mender-client_latest_${arch}.deb;
       done
+
+publish:s3:scripts:install-mender-sh:
+  stage: publish
+  dependencies:
+    - test:acceptance
+  image: debian:buster
+  before_script:
+    - apt update && apt install -yyq awscli
+  script:
+    - echo "Publishing install-mender.sh to S3"
+    - aws s3 cp scripts/install-mender.sh s3://$S3_BUCKET_NAME/$S3_BUCKET_SCRIPTS_PATH/install-mender.sh
```

We need  to set public access explicitly, see other publish jobs above.
```suggestion
    - aws s3 cp scripts/install-mender.sh s3://$S3_BUCKET_NAME/$S3_BUCKET_SCRIPTS_PATH/install-mender.sh
    - aws s3api put-object-acl --acl public-read --bucket ${S3_BUCKET_NAME}
          --key latest/${S3_BUCKET_SCRIPTS_PATH}/install-mender.sh
```